pom.xml

```diff
@@ -44,47 +43,44 @@
     <developers>
         <developer>
             <name>Anastasia Braginsky</name>
-            <email>sanastas@oath.com</email>
+            <email>sanastas@yahooinc.com</email>
```

Comment:
May be to remove people who are not in the company?

Reply:
I'm not sure what is common in this case. To remove non-active developers, or leave them for the credit.